app/views/students/edit.html.erb

```diff
@@ -2,15 +2,15 @@
 
 <%= form_with url:"/students/#{@student.id}", method: :patch, local: true do |form| %>
```

Comment:
This must be the prefilled out form you were asking Mike and Jamison about.  Will be interested to see what they say about it.

Reply:
Yes, will edit per Jamison's suggestions and send an update shortly!